statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -3168,7 +3168,11 @@ def test_serial_correlation(self, method, lags=None):
             After 0.12 the default maxlag will change to min(10, nobs // 5) for
             non-seasonal models and min(2*m, nobs // 5) for seasonal time
             series where m is the seasonal period.
-
+        df_adjust : bool, optional
+            If True, the degrees of freedom consumed by the model is subtracted
+            from the degrees-of-freedom used in the test so that the adjusted
+            dof for the statistics are lags - model_df. In an ARMA model, this
+            value is usually p+q where p is the AR order and q is the MA order.
```

Comment:
Should add the text:

When using df_adjust, it is not possible to use tests based on fewer than model_df lags. 

(This is to explain to users why they will get NaN values).

Reply:
Done.